packages/ipfs/src/http/utils/multipart-request-parser.js

```diff
@@ -92,6 +79,21 @@ async function * parseEntry (stream, options) {
     }
 
     const disposition = parseDisposition(part.headers['content-disposition'])
+    const query = qs.parse(disposition.name.split('?').pop())
```

Comment:
is go implementation ok with added query params ? As in is it going to ignore them ?

Reply:
I've added a test for this, it ignores them.